ntbs-service/DataMigration/CaseManagerImportService.cs

```diff
@@ -33,28 +33,48 @@ public class CaseManagerImportService : ICaseManagerImportService
             _logger = logger;
         }
         
-        public async Task ImportOrUpdateCaseManager(Notification notification,
+        public async Task ImportOrUpdateCaseManagersFromNotificationAndTreatmentEvents(Notification notification,
             PerformContext context,
             string requestId)
         {
-            var legacyCaseManager = await GetLegacyNotificationCaseManager(notification.LegacyId);
-            if (legacyCaseManager == null)
+            var legacyNotificationCaseManager = await GetLegacyNotificationCaseManager(notification.LegacyId);
+            if (legacyNotificationCaseManager == null)
             {
                 return;
             }
```

Comment:
Is it possible for a legacy notification to have no case manager, but for treatments events of the notification to have a case manager?

If it is then we don't want to return here.

Reply:
Yep could happen, have now inverted the if